src/constants/tax.ts

```diff
@@ -0,0 +1,75 @@
+import { ChainId, Currency, Percent } from '@uniswap/sdk-core'
+
+import { ZERO_PERCENT } from './misc'
+
+interface TokenTaxMetadata {
+  buyFee?: Percent
+  sellFee?: Percent
```

Comment:
It's weird to me that this is called `buyFee`/`sellFee` internally (to the file), but exported as `getInputTax`/`getOutputTax`.